docs/examples/simple_les.rst

```diff
@@ -0,0 +1,6 @@
+.. _benchmarks_examples:
```

Comment:
We can either have a section with multiple small examples here or just this one example detailed, then link to the input files of the other examples. I think the former is preferable, because we will never have time to draft a nice step by step description of all benchmark tests